src/components/FilterBar/FilterBar.stories.tsx

```diff
@@ -72,6 +73,90 @@ const multiSelectItems = [
 const shortMultiSelectItems = [...multiSelectItems];
 shortMultiSelectItems.pop();
 
+const meta: Meta<typeof FilterBar> = {
+  title: "Components/Form Elements/FilterBar",
+  component: FilterBar,
+  decorators: [withDesign],
+  argTypes: {
+    children: { table: { disable: true } },
+    id: { control: false },
+    selectedItems: { control: false },
+    showClearAll: {
+      type: { name: "boolean", required: false },
+      table: { defaultValue: { summary: "false" } },
+    },
+    showSubmitAll: {
+      type: { name: "boolean", required: false },
+      table: { defaultValue: { summary: "false" } },
+      description:
+        "If passed the `Apply Filters` button will render on desktop and tablet `boolean`",
+    },
+    onSubmit: {
+      // type: { name: "() => void", required: false },
+      description:
+        "Function to handle submit of all selectedItems - needs to be provided from consuming app",
+    },
+    onClear: {
+      // type: { name: "() => void", required: false },
+      description:
+        "Function to clear all selectedItems - available through hook",
+    },
+    onOpen: {
+      contol: false,
+      description:
+        "Function to open the filter modal overlay on mobile viewport - available through hook",
+      // type: { name: "() => void", required: false },
+    },
+    onClose: {
+      control: false,
+      description:
+        "Function to close the filter modal overlay on mobile viewport - available through hook",
+      // type: { name: "() => void", required: false },
+    },
+    onToggle: {
+      control: false,
+      description:
+        "Function to toggle the filter modal overlay on mobile viewport - available through hook",
+      // type: { name: "() => void", required: false },
+    },
+  },
+};
+
+export default meta;
+type Story = StoryObj<typeof FilterBar>;
+
+/**
+ * Main Story for the FilterBar component. This must contains the `args`
+ * and `parameters` properties in this object.
+ */
+export const WithControls: Story = {
+  args: {
+    id: "filterbar-id",
+    headingText: "FilterBar",
+    layout: "row",
+  },
+  render: (args) => <FilterBarStory {...args} />,
+  parameters: {
+    design: {
+      type: "figma",
+      url: "https://www.figma.com/file/qShodlfNCJHb8n03IFyApM/Main?node-id=53603%3A41770&t=j06bIokU6KAF2JHN-0",
+    },
+    jest: ["FilterBar.test.tsx"],
+  },
+};
+
+// The following are additional FilterBar example Stories.
+export const LayoutPatterns: Story = {
+  render: () => <FilterBarLayoutStory />,
+};
+export const UIContainers: Story = {
```

Comment:
The names of the `UI Container` stories need more clarity.  Add the following for this srtory:
```
name: "UI Containers for Row Layout";
```
You might even consider changing the name of the Sotry const to something more applicable.